src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Components/SearchController.cs

```diff
@@ -140,7 +134,11 @@ private List<SearchResult> AllOpenWindows(List<Window> openWindows)
             }
         }
 
-        return result.OrderBy(w => w.Result.Title).ToList();
+        return SettingsManager.Instance.InMruOrder
+            ? result.ToList()
```

Comment:
Wow. It was literally already in order (I see what you were saying over chat). That is hilarious to me.

Reply:
isnt it? It's literally less work to put in order